notifications-backend/pom.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>notifications-backend</artifactId>
+
+    <parent>
+        <groupId>com.redhat.cloud.notifications</groupId>
+        <artifactId>notifications</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+    </parent>
+
+    <properties>
+        <mockserver-client-java.version>5.5.4</mockserver-client-java.version> <!-- not the newest, but matches what is in org.testcontainers:mockserver -->
+        <version.cloud-commons>0.1.1</version.cloud-commons>
+        <openapi-parser.version>4.0.4</openapi-parser.version>
+        <maven-exec-plugin.version>3.0.0</maven-exec-plugin.version>
+    </properties>
```

Comment:
These could be in the parent pom.

Reply:
Does it make sense? We only used in the child project. I would keep it "as private" as possible and move it to the parent pom when we use it in the other project as well. Would this be okay for you?

Reply:
Well, I had bad multi-modules migrations experiences in the past because of versions declared at module level so I'm more in favor of centralized versions management at parent level now 😄

Reply:
I just moved the properties in the parent pom so you can sleep well today :D